internal/store/persistentvolume.go

```diff
@@ -32,11 +32,49 @@ import (
 )
 
 var (
+	descPersistentVolumeClaimRefName          = "kube_persistentvolume_claimref_info"
+	descPersistentVolumeClaimRefHelp          = "PersistentVolume spec.claimRef converted to Prometheus labels."
```

Comment:
Let's add this to the metric itself instead of here.

```suggestion
	descPersistentVolumeClaimRefHelp          = "Information about the Persitant Volume Claim Reference."
```